crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -607,6 +714,8 @@ pub(crate) fn removed_in_3(checker: &mut Checker, expr: &Expr) {
             if let Some(qualname) = checker.semantic().resolve_qualified_name(func) {
                 removed_argument(checker, &qualname, arguments);
             };
+
+            removed_method(checker, func);
```

Comment:
Can we skip calling `removed_method` if the `resolved_qualified_name` call was successful or does it need both?

Reply:
I think both are needed.